README.md

```diff
@@ -13,14 +13,21 @@ CC-BY 4.0: <https://github.com/twitter/twemoji/blob/gh-pages/LICENSE-GRAPHICS>
 ## Contributing
 When submitting PRs keep the following in mind please:
 
-- only submit SVGs.
+- only submit SVGs. PNGs will be automatically generated after a merged PR.
```

Comment:
are they?
I dont think we have a GH Action for that
maybe change to something like "PNGs will be generated by the maintainer after some merges"